CONTRIBUTING.md

```diff
@@ -36,5 +36,4 @@ Use the following steps when creating a new pull request:
 1. Create a local fork of this repository with your changes.
 2. Test that your changes build locally by running `swift package plugin generate-documentation --target TSPL --transform-for-static-hosting`.
 3. Create a pull request in this repository.
-4. Add @amartini51 and @krilnon as reviewers.
-5. Confirm that your changes build in CI by commenting `@swift-ci please test` on your pull request.
+4. Confirm that your changes build in CI by commenting `@swift-ci please test` on your pull request.
```

Comment:
Only users with commit access will be able to trigger CI. The phrasing here could ask you to reach out to someone with commit access to verify your changes.

Reply:
Good to know.  It sound like the old step 4 and 5 should be combined into something like:

> During the review, someone with commit access will assign reviewers and build your changes in CI.

Reply:
Changed in a94eb024.